products/silverstripe.md

```diff
@@ -74,6 +74,41 @@ releases:
     latest: "4.10.11"
     latestReleaseDate: 2022-11-21
 
+-   releaseCycle: "3.7"
+    releaseDate: 2018-06-08
+    support: true
+    eol: 2021-09-30
+    latest: "3.7.7"
+    latestReleaseDate: 2021-06-02
+
+-   releaseCycle: "3.1"
+    releaseDate: 2013-10-01
+    support: true
+    eol: 2016-12-31
+    latest: "3.1.21"
+    latestReleaseDate: 2016-11-18
+
+-   releaseCycle: "3.0"
+    releaseDate: 2012-06-28
+    support: true
+    eol: 2015-10-12
```

Comment:
Where can I confirm this date ?

Reply:
As stated in the blog post https://www.silverstripe.org/blog/silverstripe-2-4-end-of-life-announcement/ the release policy was at the time that support lasts for 2 minor versions. So end of support for 3.2 was on the date of the release of 3.2 (https://github.com/silverstripe/silverstripe-framework/releases/tag/3.2.0)